llarp/service/outbound_context.cpp

```diff
@@ -271,6 +271,17 @@ namespace llarp
       if (m_LookupFails > 16 || m_BuildFails > 10)
         return true;
 
+      constexpr auto InboundTrafficTimeout = 5s;
```

Comment:
maybe not the best place for such a const to be declared?

Reply:
i mean, it's used in only 1 place right now and making a header for all tunable parameters seems messier.